File: week-03/day-1/38.py
# Write a program that prints the numbers from 1 to 100.
# But for multiples of three print "Fizz" instead of the number
# and for the multiples of five print "Buzz".
# For numbers which are multiples of both three and five print "FizzBuzz".
for i in range(1, 101):
    printable = ''
    if i % 3 == 0:
        printable += 'Fizz'
    if i % 5 == 0:
        printable += 'Buzz'
    if printable == '':
        printable = i
    print(printable)

File: week-07/day-1/38.js
'use strict';

// Write a program that prints the numbers from 1 to 100.
// But for multiples of three print "Fizz" instead of the number
// and for the multiples of five print "Buzz".
// For numbers which are multiples of both three and five print "FizzBuzz".
for (let i = 1; i < 101; i++) {
  let output = '';
  if (i % 3 === 0) {
    output = 'Fizz';
  }
  if (i % 5 === 0) {
    output += 'Buzz';
  }
  if (output === '') {
    output = String(i);
  }
  console.log(output);
}
